bindings/pyroot_experimental/cppyy/CPyCppyy/CMakeLists.txt

```diff
@@ -93,7 +92,11 @@ foreach(val RANGE ${how_many_pythons})
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
   )
 
-  target_link_libraries(${libname} cppyy_backend${python_under_version_string} ${python_library})
+  if(APPLE)
+    target_link_libraries(${libname} PUBLIC -Wl,-bind_at_load -Wl,-undefined -Wl,dynamic_lookup cppyy_backend${python_under_version_string})
+  elseif(NOT MSVC)
+    target_link_libraries(${libname} PUBLIC -Wl,--unresolved-symbols=ignore-all cppyy_backend${python_under_version_string})
```

Comment:
Same comment for @bellenot here (equivalent flags for Windows). If this can't be done on Windows, we can just fall back to not build with unresolved symbols on Windows.

Reply:
Same comment here